ops/demo/alerts.tf

```diff
@@ -3,8 +3,14 @@ module "metric_alerts" {
   env                 = local.env
   app_service_plan_id = module.simple_report_api.app_service_plan_id
   app_service_id      = module.simple_report_api.app_service_id
-  action_group_id     = data.terraform_remote_state.global.outputs.slack_alert_action_id
   severity            = 1
   rg_name             = data.azurerm_resource_group.rg.name
   tags                = local.management_tags
+  mem_threshold       = 85
+
+  action_group_ids = [
```

Comment:
I think we need a better way to do this than a straight list. There are probably many things we'd like to alert on that don't require paging someone, but with this configuration we wouldn't be able to tell which `id` is which.

Reply:
Based on the discussion below, I think we'll actually end up with a single-item list in the short term (since cutting out our direct slack hook and relying on PD seems saner)—probably we can customize the event payload if we want pagerduty to handle individual alerts differently or we can have high-priority and low-priority services, but I'd prefer to wait until we know what our needs are before trying to do anything more complicated.

Reply:
It's more that we should decide if we ever want Slack alerts without PagerDuty alerts or vice-versa. With a straight list we can't do that because we won't be able to tell which `id` is which.

Reply:
PagerDuty can send alerts just to Slack. A service can be configured to not send texts, app notifications, etc, and only publish via Slack or whatever medium. That's why I'm suggesting we don't need the Slack integration directly in our Terraform. We can do it through PagerDuty.